demo/riskmodels/fraud_detection/spark_lgbm.py

```diff
@@ -0,0 +1,204 @@
+    subprocess.run(['aws', 's3', 'cp', 'lightgbm.onnx', model_onnx_path], cwd='./')
+    print(type(onnx_model))
+    print(type(loaded_model))
+    return loaded_model
+
+def load_onnx(onnx_ml):
+    onnx_ml = onnx_ml.setDeviceType("CPU") \
+                     .setFeedDict({"input": "features"}) \
+                     .setFetchDict({"probability": "probabilities", "prediction": "label"}) \
+                     .setMiniBatchSize(5000)
+    return onnx_ml
+
+def launch_onnx_model(loaded_model):
+    from synapse.ml.onnx import ONNXModel
+    onnx_ml = ONNXModel().setModelPayload(loaded_model.SerializeToString())
+    print("Model inputs:" + str(onnx_ml.getModelInputs()))
+    print("Model outputs:" + str(onnx_ml.getModelOutputs()))
+    print("Model type:" )
+    print(type(onnx_ml))
+    return onnx_ml
+
+def onnx_transform(onnx_ml, test_data):
+    result = onnx_ml.transform(test_data)
+    print("ONNX transform sample:", result.limit(10).toPandas())
+
+if __name__=="__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--conf', type=str, action='store', default='', help='config file path')
+    args = parser.parse_args()
+    params = load_config(args.conf)
+    spark = init_spark(**params)
+
+    ## get train and test data
+    train_dataset, test_dataset = read_dataset(spark, **params)
+    ## feature transformation
+    feature_cols, label_col = get_feature_and_label_cols(train_dataset, **params)
+    train_data = get_vectorassembler(train_dataset, feature_cols=feature_cols, label=label_col)
+    print("Debug -- train input features:")
+    train_data.show(10, False)
+    test_data = get_vectorassembler(test_dataset, feature_cols=feature_cols, label=label_col)
+    print("Debug -- test input features:")
+    test_data.show(10, False)
+    ## fit model and test
+    model = train(spark, train_data, label_col, **params['model_params'])
+
+    ## eval the train dataset
+    print("Debug -- train sample prediction:")
+    predictions = model.transform(train_data)
+    predictions.show(10, False)
+    auc, ks = evaluate(spark, predictions, label_col)
+    print("Debug -- train auc:", auc)
+    print("Debug -- train ks statistic:", ks.statistic)
+    ## eval the test dataset
+    print("Debug -- test sample prediction:") 
+    predictions = model.transform(test_data)
+    predictions.show(10, False)
+    auc, ks = evaluate(spark, predictions, label_col)
+    print("Debug -- test auc:", auc)
+    print("Debug -- test ks statistic:", ks.statistic)
+
+    ## write evaluation to s3
+    write_dataset_to_s3(predictions, **params)
+    ## convert model to onnx format & load the model
+    print("Debug -- transform lightgbm model into ONNX format...") 
+    onnx_model = get_onnx_model(model,len(train_dataset.columns)-1)
+    loaded_model = save_onnx_model(onnx_model, **params)
+    onnx_ml = launch_onnx_model(loaded_model)
+    ## predicting using onnx lightgbm model
+    onnx_ml = load_onnx(onnx_ml)
```

Comment:
Merge load_onnx and launch_onnx_model into one single func.

Reply:
fixed